AdSec/Python/requirements.txt

```diff
@@ -1 +1,3 @@
 oasys.adsec
+svglib==1.3.0

```

Comment:
Do we have a ticket to update the examples to work with the latest version of svglib?

Reply:
Hi Tim, I have created a ticket now, https://ovearup.atlassian.net/browse/ADSEC-1539